src/utils/fetchTransactions.ts

```diff
@@ -1,4 +1,5 @@
 import { ethers } from 'ethers'
+import { Block as IBlock } from 'interfaces/block.interface'
 import { Transaction as TransactionI } from 'interfaces/transaction.interface'
```

Comment:
suggestion(non-blocking): we should probably be consistent here with the naming. I would suggest to name interfaces as `I{MODEL_NAME}` -> should be done directly in the interface file